pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -6,8 +6,8 @@ exp,benchmark
 15,AlterTableAddCheckConstraint/alter_table_add_2_check_constraints
 15,AlterTableAddCheckConstraint/alter_table_add_3_check_constraints
 15,AlterTableAddColumn/alter_table_add_1_column
-15,AlterTableAddColumn/alter_table_add_2_columns
-15,AlterTableAddColumn/alter_table_add_3_columns
+19,AlterTableAddColumn/alter_table_add_2_columns
```

Comment:
Investigate this by adding a data-driven scbuild test. Here's what's going on:
```> EXPLAIN (DDL, SHAPE) ALTER TABLE alter_table ADD COLUMN a INT;                                                                
                                            info
--------------------------------------------------------------------------------------------
  Schema change plan for ALTER TABLE ‹defaultdb›.public.‹alter_table› ADD COLUMN ‹a› INT8;
   └── execute 3 system table mutations transactions
```
and
```
> EXPLAIN (DDL, SHAPE) ALTER TABLE alter_table ADD COLUMN a INT, ADD COLUMN b INT;                                              
                                                      info
-----------------------------------------------------------------------------------------------------------------
  Schema change plan for ALTER TABLE ‹defaultdb›.public.‹alter_table› ADD COLUMN ‹a› INT8, ADD COLUMN ‹b› INT8;
   ├── execute 2 system table mutations transactions
   ├── backfill using primary index alter_table_pkey- in relation alter_table
   │    └── into alter_table_pkey+ (rowid, a+, b+)
   ├── execute 2 system table mutations transactions
   ├── merge temporary indexes into backfilled indexes in relation alter_table
   │    └── from alter_table@[3] into alter_table_pkey+
   ├── execute 1 system table mutations transaction
   ├── validate UNIQUE constraint backed by index alter_table_pkey+ in relation alter_table
   └── execute 3 system table mutations transactions
```